astronomer/providers/amazon/aws/hooks/batch_client.py

```diff
@@ -0,0 +1,256 @@
+import asyncio
+from random import sample
+from typing import Any, Dict, List, Optional, Union
+
+import botocore.exceptions
+from airflow.exceptions import AirflowException
+from airflow.providers.amazon.aws.hooks.batch_client import BatchClientHook
+
+from astronomer.providers.amazon.aws.hooks.base_aws_async import AwsBaseHookAsync
+
+
+class BatchClientHookAsync(BatchClientHook, AwsBaseHookAsync):
+    """
+    Async client for AWS Batch services.
+
+    :param max_retries: exponential back-off retries, 4200 = 48 hours;
+        polling is only used when waiters is None
+
+    :param status_retries: number of HTTP retries to get job status, 10;
+        polling is only used when waiters is None
+
+    .. note::
+        Several methods use a default random delay to check or poll for job status, i.e.
+        ``random.sample()``
+        Using a random interval helps to avoid AWS API throttle limits
+        when many concurrent tasks request job-descriptions.
+
+        To modify the global defaults for the range of jitter allowed when a
+        random delay is used to check Batch job status, modify these defaults, e.g.:
+        .. code-block::
```

Comment:
It will come like below...similar to apache airflow documentation

<img width="723" alt="image" src="https://user-images.githubusercontent.com/94376113/171573769-41567835-0fa8-4634-a00c-4f768a36a9e2.png">

Reply:
ok, so the code block does not work inside the note block and it just acts like a string. so maybe you can remove `.. code-block:: python`

Reply:
addressed in [e10446b](https://github.com/astronomer/astronomer-providers/pull/404/commits/e10446b11b34f47b670ecf8f77f19471ac292221)